src/ApiDescriptionChecker.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenApiGenerator;
 
+use OpenApiGenerator\Exceptions\DefinitionCheckerException;
+
 class ApiDescriptionChecker
```

Comment:
maybe rename to ValidatorSchema?

Reply:
Definitely ! I just won't be able to review this before a couple hours

Reply:
So, a couple of hours have passed?